src/main/java/de/tum/in/www1/exerciseapp/domain/Exercise.java

```diff
@@ -24,7 +24,7 @@
     discriminatorType = DiscriminatorType.STRING
 )
 @DiscriminatorValue(value = "E")
-@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
```

Comment:
why is this change necessary?

Reply:
Turns out, its not necessary, I will revert it.